packages/sdk/src/riverConfig.ts

```diff
@@ -47,14 +51,14 @@ function makeWeb3Deployment(environmentId: string): Web3Deployment {
         return getWeb3Deployment(environmentId)
     }
     // Fallback to env vars
-    check(isDefined(process.env.BASE_CHAIN_ID), 'BASE_CHAIN_ID is not defined')
-    check(isDefined(process.env.BASE_CHAIN_RPC_URL), 'BASE_CHAIN_RPC_URL is not defined')
-    check(isDefined(process.env.BASE_REGISTRY_ADDRESS), 'BASE_REGISTRY_ADDRESS is not defined')
-    check(isDefined(process.env.SPACE_FACTORY_ADDRESS), 'SPACE_FACTORY_ADDRESS is not defined')
-    check(isDefined(process.env.SPACE_OWNER_ADDRESS), 'SPACE_OWNER_ADDRESS is not defined')
-    check(isDefined(process.env.RIVER_CHAIN_ID), 'RIVER_CHAIN_ID is not defined')
-    check(isDefined(process.env.RIVER_CHAIN_RPC_URL), 'RIVER_CHAIN_RPC_URL is not defined')
-    check(isDefined(process.env.RIVER_REGISTRY_ADDRESS), 'RIVER_REGISTRY_ADDRESS is not defined')
+    check(isDefined(process?.env?.BASE_CHAIN_ID), 'BASE_CHAIN_ID is not defined')
+    check(isDefined(process?.env?.BASE_CHAIN_RPC_URL), 'BASE_CHAIN_RPC_URL is not defined')
+    check(isDefined(process?.env?.SPACE_FACTORY_ADDRESS), 'SPACE_FACTORY_ADDRESS is not defined')
+    check(isDefined(process?.env?.BASE_REGISTRY_ADDRESS), 'BASE_REGISTRY_ADDRESS is not defined')
+    check(isDefined(process?.env?.SPACE_OWNER_ADDRESS), 'SPACE_OWNER_ADDRESS is not defined')
+    check(isDefined(process?.env?.RIVER_CHAIN_ID), 'RIVER_CHAIN_ID is not defined')
+    check(isDefined(process?.env?.RIVER_CHAIN_RPC_URL), 'RIVER_CHAIN_RPC_URL is not defined')
+    check(isDefined(process?.env?.RIVER_REGISTRY_ADDRESS), 'RIVER_REGISTRY_ADDRESS is not defined')
```

Comment:
can you pop this into its own commit?

Reply:
https://github.com/river-build/river/pull/279